src/content/app/genome-browser/components/drawer/components/sequence-view/TranscriptSequenceView.test.tsx

```diff
@@ -173,30 +173,28 @@ describe('<TranscriptSequenceView />', () => {
         transcript: proteinCodingTranscript
       });
 
-      waitFor(() => {
-        const sequenceContainer = container.querySelector(
-          '.sequence'
-        ) as HTMLElement;
-        expect(sequenceContainer.textContent).toBe(mockGenomicSequence);
+      await waitFor(() => {
+        const sequenceContainer = container.querySelector('.sequence');
+        expect(sequenceContainer?.textContent).toBe(mockGenomicSequence);
       });
 
       const cdsSequenceLabel = getByLabelText('CDS');
 
       await userEvent.click(cdsSequenceLabel);
 
-      waitFor(() => {
-        const sequenceContainer = container.querySelector(
-          '.sequence'
-        ) as HTMLElement;
-        expect(sequenceContainer.textContent).toBe(mockCDSSequence);
+      await waitFor(() => {
+        const sequenceContainer = container.querySelector('.sequence');
+        expect(sequenceContainer?.textContent).toBe(mockCDSSequence);
       });
     });
   });
 
   describe('reverse complement', () => {
     it('only shows the reverse complement checkbox for the genomic sequence', async () => {
       const { getByLabelText, queryByLabelText } = renderTranscriptSequenceView(
-        { transcript: proteinCodingTranscript }
+        {
+          transcript: proteinCodingTranscript
+        }
```

Comment:
Has prettier changed its default rules again? 👀

Reply:
Prettier doesn't seem to allow thiis:

```
      const { getByLabelText, queryByLabelText } = renderTranscriptSequenceView({
          transcript: proteinCodingTranscript
      });
```
because `      const { getByLabelText, queryByLabelText } = renderTranscriptSequenceView({` is exactly 81 characters.

I hate prettier! Almost as much as my Mac's keyboard.